site/checklist_fr.md

```diff
@@ -0,0 +1,149 @@
+ - [ ] Fournissez-vous les mêmes types d'informations et de descriptions lorsque vous écrivez sur des personnes de genres différents ?([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Utilisez-vous « ils » comme une option neutre en termes de genre ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+
+#### Inclusion Culturelle
+- [ ] Éviter l'appropriation culturelle ([ref](https://au.reachout.com/articles/why-cultural-appropriation-isnt-cool))
+    - [ ] pas ~~Gourou~~ → **expert**, **enseignant** ([ref](https://www.dictionary.com/e/stop-using-in-2020?utm_medium=referral&utm_source=idealist))
+    - [ ] pas ~~Animal spirituel~~ → **mascotte personnelle** ([ref](https://www.selfdefined.app/definitions/pow-wow/))
+    - [ ] pas ~~Pow-wow~~ → si tu voulais dire "réunion", [regardes ces synonymes](https://www.thesaurus.com/browse/meeting)). ([ref](https://www.selfdefined.app/definitions/pow-wow/))
+    - [ ] pas ~~Tribu~~ → s'abstenir de l'utiliser, ou le remplacer par un terme sans appropriation comme "groupe d'amis proches" ([ref](https://www.selfdefined.app/definitions/tribe/))
+    - [ ] pas ~~Animal Totem~~ → s'abstenir de l'utiliser ([ref](https://www.selfdefined.app/definitions/totemic-animal/))
+- [ ] Évitez les termes et expressions culturellement offensants
+    - [ ] pas ~~"Indien donneur"~~ → s'abstenir de l'utiliser ([ref](https://www.ictinc.ca/blog/culturally-offensive-phrases-you-should-use-at))
+    - [ ] pas ~~"Trop de chefs et pas assez d'Indiens"~~ → s'abstenir de l'utiliser ([ref](https://www.ictinc.ca/blog/culturally-offensive-phrases-you-should-use-at))
+    - [ ] pas ~~"Faites le tour des wagons", "Tenir le fort", "Sur le sentier de la guerre"~~ → s'abstenir de les utiliser ([ref](https://www.ictinc.ca/blog/culturally-offensive-phrases-you-should-use-at))
+    - [ ] pas ~~"Indien donneur"~~ → s'abstenir de l'utiliser ([ref](https://www.ictinc.ca/blog/indian-giver-come-can-give-back))
+    - [ ] pas ~~"Heure indienne", "Été indien"~~ → s'abstenir de l'utiliser ([ref](https://www.ictinc.ca/blog/culturally-offensive-phrases-you-should-use-at))
+    - [ ] pas ~~"Gitan / Juif"~~ → **arnaqué**, **agrippé** ([ref](http://meloukhia.net/2011/02/language_matters_no_gypped_is_pas_a_good_alternative_to_jewed/))
+    - [ ] pas ~~"Gitan"~~ → s'abstenir de l'utiliser ([ref](https://now.org/blog/the-g-word-isnt-for-you-how-gypsy-erases-romani-women/))
+    - [ ] pas ~~"Karma"~~ → **châtiment** ([ref bientôt disponible](A FAIRE))
+
+#### Inclusion du Handicap
+- [ ] Éviter l'inspiration porno ([ref](https://www.youtube.com/watch?v=SxrS7-I_sMQ))
+- [ ] Lorsque vous discutez d'un produit, indiquez ce qui le rend accessible ([ref bientôt disponible](A FAIRE))
+- [ ] Éviter le langage capacitiste ([ref](http://web.augsburg.edu/english/writinglab/Avoiding_Ableist_Language.pdf)), ([ref](https://www.unr.edu/nevada-today/blogs/2020/avoiding-ableist-terminology))
```

Comment:
I think that the equivalent of "ableism" in french is "Validisme" - against, not yet as common.

"Eviter le language capacististe/le validisme